src/main/java/io/supertokens/oauth/OAuth.java

```diff
@@ -386,4 +467,41 @@ public static JsonObject introspectAccessToken(Main main, AppIdentifier appIdent
         result.addProperty("active", false);
         return result;
     }
+
+    public static void revokeAllConsentSessions(Main main, AppIdentifier appIdentifier, Storage storage, String subject, String clientId) throws StorageQueryException {
+        OAuthStorage oauthStorage = StorageUtils.getOAuthStorage(storage);
+        if (clientId == null) {
+            oauthStorage.revoke(appIdentifier, "sub", subject);
+        } else {
+            oauthStorage.revoke(appIdentifier, "client_id_sub", clientId + ":" + subject);
+        }
+    }
+
+	public static void revokeRefreshToken(Main main, AppIdentifier appIdentifier, Storage storage, String token) throws StorageQueryException, NoSuchAlgorithmException {
+		OAuthStorage oauthStorage = StorageUtils.getOAuthStorage(storage);
```

Comment:
Does this propagate to hydra? Or does this prevent future `refresh_token` grant flows from working?

Reply:
Fixed